tests/pw/README.MD

```
@@ -1,389 +1,574 @@
 
-## :bookmark: Table of contents
+## Table of Contents
 
-- [Pre-requisites :pushpin:](#pre-requisites)
-    - [Install Node](#install-node)
-    - [Install NVM](#install-nvm)
-    - [Install Docker](#install-docker)
+- [Overview](#overview)
+- [Prerequisites](#prerequisites)
+- [Quick Start](#quick-start)
+- [Configuration](#configuration)
```

markdownlint-cli2 (0.20.0)

[warning] Link fragments should be valid

(MD051, link-fragments)